### DISCUSSION TITLE
Leistung Zeitlich begrenzen um schedule Funktion erweitern?

### DISCUSSION THREAD

**Discussion by testpaul999** [tbnobody/OpenDTU#796] [Ideas]
Guten Tag, 
ich würde es gut finden, wenn die Funktion Leistung zeitlich begrenzen erweitert werden könnte. 
Ich stelle mir vor von sechs bis 10:00 Uhr 100 % von 10:00 Uhr bis 18:00 Uhr 50 % ab 18:00 Uhr wieder 100 %.

Danke

**Comment by helgeerbe** [comment]
Kannst du vielleicht auch erklären warum?

**Reply by testpaul999** [reply]
Hallo, 
ich nutze den Hoymiles nur als zusätzliche Quelle zu meinem ESS Victron System.
Da das Victron ESS aber erst 3% in den Akku laden muss, habe ich somit auch schon Einspeisung ins Hausnetz.
Wenn dann aber Mittags die Sonne voll da ist, der Akku aber noch nicht voll, Speise ich den Überschuss dann ein.
Das würde ich gerne reduzieren, mehr nicht.

Grüße

**Reply by DPO99** [reply]
So eine Steuerung ist z.B. über IoBroker oder HomeAssistand oder über einen MQTT Broker super einfach einzubauen.

**Reply by testpaul999** [reply]
Guten Tag, 
Ja wahrscheinlich, wenn man Ahnung hat.
Aber ich in auch kein Fan, noch ein zusätzliches Device mit einzubauen.
Verbindung weg, Reglung weg…..
Grüße

**Reply by mastertape** [reply]
genau das ist die sache... fände es auch super wenn so ne art tag und nachtprofil gehen würde

**Reply by resonnbas** [reply]
Hallo Mastertape,
im Beitrag" Uhrzeitbasiertes Limit" wird schon wild diskutiert. Aber leider kein wirkliches gehör gefunden . Dieser Beitrag ist ein Vorläufer davon. Bitte mache doch im anderen Beitrag mit damit es evtl ein Erbarmen bei den Entwicklern gibt. Danke.  Grüße Rene

**Comment by Matthias84** [comment]
Mein konkreter Anwendungsfall ist etwas anders gelagert: Durch eine relativ lange Zuleitung und Nachbarn mit größeren PV-Anlagen komme ich regelmäßig über die Mittagszeit in eine Notabschaltung, weil die Phase über eine längere Zeit >250V hat und so der Mikroinverter abschaltet.

Bei geringerer Leistung tritt das Phänomen gerade zur Mittagszeit weniger oft auf. Für mich wäre also eine zeitlich begrenzte max. Leistung sehr hilfreich. Klar ließe sich das mit homeassistant & Co auch bewerkstelligen, aber sowas muss dazu erst einmal laufen ;-)
Mir ist noch nicht klar, wie die "temporär beschränken" Funktion in den Leistungseinstellungen funktioniert. Ist das evtl. schon für solche Sachen vorgesehen?
